libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -153,28 +133,25 @@ class CodeTransform: public boost::static_visitor<>
 			},
 			[=](Scope::Function&)
 			{
-				solAssert(false, "Not yet implemented");
+				solAssert(false, "Function not removed during desugaring.");
 			}
 		)))
 		{
+			return;
 		}
-		else if (!m_identifierAccess || !m_identifierAccess(_identifier, m_state.assembly, CodeGenerator::IdentifierContext::RValue))
-		{
-			m_state.addError(
-				Error::Type::DeclarationError,
-				"Identifier not found or not unique",
-				_identifier.location
-			);
-			m_state.assembly.append(u256(0));
-		}
+		solAssert(
+			m_identifierAccess.generateCode,
+			"Identifier not found and no external access available."
+		);
+		m_identifierAccess.generateCode(_identifier, IdentifierContext::RValue, m_state.assembly);
 	}
 	void operator()(FunctionalInstruction const& _instr)
 	{
 		for (auto it = _instr.arguments.rbegin(); it != _instr.arguments.rend(); ++it)
 		{
 			int height = m_state.assembly.deposit();
 			boost::apply_visitor(*this, *it);
-			expectDeposit(1, height, locationOf(*it));
+			expectDeposit(1, height);
```

Comment:
Do these expected deposit numbers match the `valueSize` estimated in the analysis?  Is there a way to check that when the compiler is running?